calc-duration.rb

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
I think you left this file in the PR by accident?

Most of the contents here would be better as tests.

Reply:
Removed.